BeerOverflowWindowsApp/MainWindow.cs

```diff
@@ -121,18 +120,25 @@ private void ClearSelection()
                 }
                 result.RemoveDuplicates();
                 result.RemoveBarsOutsideRadius(radius);
-                await Task.Run(() => result = new DatabaseManager().GetAllBarData(result));
+                //await Task.Run(() => 
+                result = (BarDataModel)WebApiAccess.GetAllBarData(result);
```

Comment:
Same here about the Task.Run(), as elsewhere. Can you re-add it, or does it not work?